pallets/poa/src/tests.rs

```diff
@@ -99,23 +106,73 @@ impl pallet_session::Config for Test {
 	type SessionHandler = TestSessionHandler;
 	type ShouldEndSession = TestSessionHandler;
 	type NextSessionRotation = ();
-	type Event = ();
+	type Event = Event;
 	type Keys = UintAuthorityId;
-	type ValidatorId = <Test as frame_system::Config>::AccountId;
+	type ValidatorId = AccountId;
 	type ValidatorIdOf = ConvertInto;
 	type WeightInfo = ();
 }
-impl Config for Test {}
+ord_parameter_types! {
+	pub const Admin: u64 = 4;
+}
+impl pallet_membership::Config for Test {
+	type Event = Event;
+	type AddOrigin = EnsureSignedBy<Admin, u64>;
+	type RemoveOrigin = EnsureSignedBy<Admin, u64>;
+	type SwapOrigin = EnsureSignedBy<Admin, u64>;
+	type ResetOrigin = EnsureSignedBy<Admin, u64>;
+	type PrimeOrigin = EnsureSignedBy<Admin, u64>;
+	type MembershipInitialized = ();
+	type MembershipChanged = ();
+	type MaxMembers = ConstU32<10>;
+	type WeightInfo = ();
+}
 
-pub const VALIDATOR: u64 = 1;
+impl Config for Test {
+	type ValidatorsSet = Membership;
+}
+
+parameter_types! {
+	pub const Validator01: AccountId = 1;
+	pub const Validator02: AccountId = 2;
+	pub const Validator03: AccountId = 3;
+}
 
 // This function basically just builds a genesis storage key/value store according to
 // our desired mockup.
-fn new_test_ext() -> sp_io::TestExternalities {
-	frame_system::GenesisConfig::default()
+pub fn new_test_ext() -> sp_io::TestExternalities {
+	sp_tracing::try_init_simple();
+
+	let mut storage = frame_system::GenesisConfig::default()
 		.build_storage::<Test>()
-		.unwrap()
-		.into()
+		.unwrap_or_else(|err| {
```

Comment:
It might be [helpful to use `expect` here](https://doc.rust-lang.org/book/ch09-02-recoverable-errors-with-result.html_, unless the explicit `panic!` is needed.

Reply:
It was using an `unwrap` and replacing an `unwrap` with a `panic` was not what I had in mind.